src/inverseRobot.js

```diff
@@ -25,6 +25,24 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const values = Object.values(robot);
+  const uniqueValues = unique(values);
+
+  if (values.length !== uniqueValues.length) {
+    return null;
+  }
```

Comment:
This code will work, but you don't need to create array, then pack it to set.
You could just check in object that you have duplicated keys